en/identity-server/next/docs/apis/organization-apis/restapis/email-templates-v2.yaml

```diff
@@ -0,0 +1,891 @@
+openapi: 3.0.0
+info:
+  version: v2
+  title: WSO2 Identity Server - Email Template API
+  description: 'This document specifies a RESTful API for **Managing Email Templates of Organizations** in WSO2 Identity Server. This API allows adding, retrieving, replacing, and removing email templates and template types.'
+tags:
+  - name: Email Template Types
+    description: An email template type can have multiple email templates for different languages.
+  - name: Email Templates
+    description: An email template of a specific type and a language.
+  - name: Application Email Templates
+    description: |
+      An application-specific email template of a specific type and a language.
+
+      Application-specific email templates are triggered whenever the user interacts with applications that use the default WSO2 Identity Server UI. However, if the user interacts directly with APIs, application-specific email templates are supported by the following APIs:
+
+      - User creation with SCIM2 APIs
+      - Locking/Unlocking accounts with SCIM2 APIs
+      - Password recovery API
+      - Email verification flows
+      - Resend email flows
+
+      Note: To ensure application-specific branding is applied during the above API interactions, they must be invoked using an access token issued specifically for the application. If an access token generated from admin credentials is used instead, the branding will default to organization-specific settings.
+security:
+  - OAuth2: []
+paths:
+  /email/template-types:
+    get:
+      tags:
+        - Email Template Types
+      summary: Retrieves all the email template types.
+      operationId: getAllEmailTemplateTypes
+      description: |
+        Retrieves all the email template types in the system. <br>
+
+        <b>Scope(Permission) required:</b> `internal_org_email_mgt_view`
+      parameters:
+        - $ref: '#/components/parameters/limitQueryParam'
+        - $ref: '#/components/parameters/offsetQueryParam'
+        - $ref: '#/components/parameters/sortOrderQueryParam'
+        - $ref: '#/components/parameters/sortByQueryParam'
+      responses:
+        '200':
+          description: Search results matching the given criteria.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/EmailTemplateType'
+        '400':
+          $ref: '#/components/responses/InvalidInput'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/ServerError'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl -X 'GET' \
+            'https://localhost:9443/t/carbon.super/o/api/server/v2/email/template-types' \
```

Comment:
```suggestion
            'https://localhost:9443/o/api/server/v2/email/template-types' \
```

Reply:
shall we remove in other samples as well?

Reply:
Fixed with https://github.com/wso2/docs-is/pull/5034/commits/b8d4ab4d4cbd1a407fdfcf0ae3287da4b691cad1